### PR TITLE
Add initial benchmark and controls for OCI Thrifty. Closes #1

### LINKED ISSUE #1: Add initial benchmark and controls for OCI Thrifty

### PR DESCRIPTION
**Pull request by khushboo9024**

### CODE REVIEW

File: controls/core.sp [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+      left join oci_identity_compartment as c on c.id = a.compartment_id;
+  EOT
+
+  tags = merge(local.core_common_tags, {
+    class = "deprecated"
+  })
+}
+
+control "core_instance_low_utilization" {
+  title         = "Core instances with low CPU utilization should be reviewed"
```
Comment (rajlearner17) [inline controls/core.sp]: Compute instances with low CPU utilization should be reviewed 
or
Core Compute ?

File: controls/core.sp [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+      left join oci_identity_compartment as c on c.id = b.compartment_id;
+  EOT
+
+  tags = merge(local.core_common_tags, {
+    class = "unused"
+  })
+}
+
+control "core_instance_long_running" {
+  title       = "Long running instances should be reviewed"
```
Comment (rajlearner17) [inline controls/core.sp]: Long running compute instances should be reviewed

File: controls/core.sp [hunk 3]
```diff
@@ -0,0 +1,393 @@
...
+      left join oci_identity_compartment as c on c.id = a.compartment_id;
+  EOT
+
+  tags = merge(local.core_common_tags, {
+    class = "unused"
+  })
+}
+
+control "core_volume_auto_tune_performance_enabled" {
+  title       = "Block volume should be enabled with auto-tune for performance"
```
Comment (rajlearner17) [inline controls/core.sp]: Block volume should be enabled with auto-tune for better performance

File: controls/database.sp [hunk 4]
```diff
@@ -0,0 +1,43 @@
+locals {
+  database_common_tags = merge(local.thrifty_common_tags, {
+    service = "database"
+  })
+}
+
+benchmark "database" {
+  title         = "Database Checks"
```
Comment (rajlearner17) [inline controls/database.sp:8]: This will be under Oracle Database i.e.
Database > Oracle Database Checks

**Review by Subhajit97 — APPROVED** [review]

**Review by rajlearner17 — APPROVED** [review]
LGTM

File: controls/mysql.sp [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  tags          = local.mysql_common_tags
+  children = [
+    control.mysql_db_system_age_90,
+    control.mysql_db_system_low_connection_count
+  ]
+}
+
+control "mysql_db_system_age_90" {
+  title       = "MySQL DB systems created over 90 days ago should be reviewed"
+  description = "MySQL DB systems created over 90 days ago should be reviewed and deleted if not required."
```
Comment (cbruno10) [inline controls/mysql.sp:21]: If there aren't any equivalent settings for reserved instances, or another cost saving measure for long running instances, I think we should remove this control. Having a DB system running for 90 days ago or more alone isn't enough to recommend a deletion.

File: controls/database.sp [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+  documentation = file("./controls/docs/database.md")
+  tags          = local.database_common_tags
+  children = [
+    control.database_autonomous_database_age_90,
+    control.database_autonomous_database_low_utilization
+  ]
+}
+
+control "database_autonomous_database_age_90" {
+  title       = "Autonomous database created over 90 days ago should be reviewed"
```
Comment (cbruno10) [inline controls/database.sp]: If there aren't any equivalent settings for reserved instances, or another cost saving measure for long running instances, I think we should remove this control. Having an autonomous database running for 90 days ago or more alone isn't enough to recommend a deletion.

File: controls/core.sp [hunk 7]
```diff
@@ -0,0 +1,397 @@
+locals {
```
Comment (cbruno10) [inline controls/core.sp]: @Subhajit97 Can you please try renaming this benchmark and its controls from core->compute and see how that looks in the hub site? Core is the TF + table name scheme we use, but Compute seems to be more common in OCI's docs, so I'm interested to see if this one is more recognizable for users.

File: controls/core.sp [hunk 8]
```diff
@@ -0,0 +1,397 @@
...
+  severity    = "low"
+
+  sql = <<-EOT
+    select
+      a.id as resource,
+      case
+        when a.time_created > current_timestamp - interval '90 days' then 'ok'
+        else 'alarm'
+      end as status,
+      a.display_name || ' created at ' || a.time_created || '.' as reason,
```
Comment (cbruno10) [inline controls/core.sp]: For this reason, and others in this benchmark (or others), can we also include the `(num_days)` as well?

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@Subhajit97 Please see comments, thanks!

Also, can you please update the default benchmarks in docs to reference the control pages on the hub site?

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**